spec/api/v1/trade_leads_spec.rb

```diff
@@ -62,9 +63,9 @@
         expect(json_response['offset']).to eq(0)
 
         results = json_response['results']
-        expect(results[0]).to eq(expected_results[1])
-        expect(results[1]).to eq(expected_results[2])
-        expect(results[2]).to eq(expected_results[3])
+        expect(results[0]).to eq(expected_results[4])
```

Comment:
For the future, what do you think of having atomic specs like:
    
    it { expect (results[0]).to eq expected_results[4] }

This might help identifying exactly what failed instead of debugging!?